n5k/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .calculator_base import N5KCalculatorBase
 from .calculator_ccl import N5KCalculatorCCL
+from .calculator_matter import N5KCalculatorMATTER
```

Comment:
I think this is repeated (see 4 lines below)

Reply:
Reverted!